docs/src/folderstruct.md

```diff
@@ -0,0 +1,55 @@
+- LICENSE: License file of `ClimaLSM`, you can read it to learn about legal practice regarding use of `ClimaLSM` open source code. 
+- Project.toml: The Julia programming language requires a `Project.toml` file to create an environment, which specify dependencies of a project as well as its version, name, authors and a unique identifier number (uuid). Every Julia registered package has a `Project.toml` file. 
+- README.md: This markdown file contains the info that you can read on [ClimaLSM GitHub web page](https://github.com/CliMA/ClimaLSM.jl)
+- bors.toml: This file ensure unit tests and other continuous integration requirements are met before merging a pull request to `ClimaLSM` main branch. 
+
+## /docs folder
+
+Julia packages are recommended to have a `\docs` folder that builds a standardised documentation following the official documentation generator for Julia: [Documenter.jl](https://documenter.juliadocs.org/stable/).
+
+The folder /docs contains:
+- a `/src` folder: It is recommended to put your markdown pages inside this folder. Each markdown file (.md extension text file) is a page accessible through the menu of the documentation. For example, `docs/src/Contributing.md` contains the text you can read on the documentation "contribution guide" menu. The path to this .md file and the name of the menu is set in the `docs/make.jl` file. 
+- a `make.jl` file: This Julia file contains your documentation website structure. Running this file will build your website pages, you can run it locally, but it is commonly built remotely via .github/workflows/docs.yml to generate the github static page hosted on the [gh-pages branch](https://github.com/CliMA/ClimaLSM.jl/tree/gh-pages). 
+
+Note: the documentation can have sub-menu. For example, `APIs` have seb-menu `ClimaLSM` which has many sub-menu... This structure is built in our current framework via a file `docs/list_of_apis.jl` in that example, which is then included in `docs/make.jl`.   
+
+## /experiments folder
+
+The `experiments` folder contains scripts to run ClimaLSM models. It contains a folder for `integrated` models and a folder for `standalone` models. It is meant to provide users with simple examples of ClimaLSM runs. The files contains meteorological inputs (such as precipitation), values for every parameters, and the domains and timestepper are specified. 
+
+For example, `/experiments/LSM/ozark/` contains:
+- ozark_domains.jl: Describes the soil domain (depth, number of layer), and the canopy (number and height of stems and leaves).
+- ozark_met_drivers_FLUXNET.jl: This files load meteorological input data from the ozark FLUXNET file, and does additional things such as spline interpolation of these drivers.
+- ozark_parameters.jl: In this file, parameters values are defined, for example the soil porosity is given a value of 0.55 m3 m-3. 
```

Comment:
here we may not want to give the specific value, because then any PR which changes that value also has to remember to update this docs page